api/XMLHttpRequest.json

```diff
@@ -1529,7 +1529,7 @@
                 "version_added": "20"
               },
               "ie": {
-                "version_added": null
+                "version_added": "10"
```

Comment:
If we're marking IE as a truthy value, we should mark Edge as 12 since Edge is a continuation off of IE.  Can we set it to 12?

Reply:
Yep good catch. I've pushed an update that sets Edge to 12.